domains/meerveld.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "FabianMeerveld",
+        "email": "fabian.meerveld@student.hu.nl"
+    },
+    "record": {
+        "A": ["20.160.193.51"]
```

Comment:
Unable to access site, can you provide a screenshot?

Reply:
HI, i just set up apache, it is for a school project and its only online during school hours. here is a screenshot of the site (it is online now) 
![image](https://github.com/is-a-dev/register/assets/144360059/1bce29e0-f847-4b75-ac07-f15f6a8285b0)

Reply:
Looks good to me.